RatS/filmtipset/filmtipset_ratings_parser.py

```diff
@@ -0,0 +1,82 @@
+import os
+import datetime
+import time
+import fileinput
+import re
+from shutil import copyfile
+from RatS.base.base_ratings_downloader import RatingsDownloader
+from RatS.filmtipset.filmtipset_site import Filmtipset
+from RatS.utils import file_impex
+
+# Request download form
+# https://www.filmtipset.se/installningar
+# <form method="post" autocomplete="off">
+#     <select id="export-type" name="export-type" placeholder="Välj dataslag" required="">
+#     <option value="betyg">Filmbetyg
+#     </option><option value="kommentarer">Filmkommentarer
+#     </option><option value="recensioner">Recensioner
+#     </option></select><br>
+#     <input type="submit" id="submit-export" name="submit-export" value="Exportera data" class="button"><br>
+# </form>
+
+class FilmtipsetRatingsParser(RatingsDownloader):
+    def __init__(self, args):
+        super(FilmtipsetRatingsParser, self).__init__(Filmtipset(args), args)
+        # TODO: due to timezone and potential racecondition, it's probably more safe to
+        #       find/search file in download folder
+        date = datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d')
+        self.downloaded_file_name = 'ft_betyg_{}.csv'.format(date)
+        self.csv_delimiter = ';'
+
+    # TODO remove when filmtipset has cleaned up thier "csv" file
+    @staticmethod
+    def _repair_line(line):
+        output = re.search(r'([0-9]{4}-[0-9]{2}-[0-9]{2}),(.*)(;[\-0-9]*;[1-5])$', line)
+        if output is None:
+            return line
+        group2 = output.group(2).replace('"', '""')
+        newline = '{};"{}"{}\n'.format(output.group(1), group2, output.group(3))
+        return newline
```

Comment:
```suggestion
        repaired_line = '{};"{}"{}\n'.format(output.group(1), group2, output.group(3))
        return repaired_line
```
(or `repaired_row`, see my other comment)